woostarter/templates/single.html

```diff
@@ -1,97 +1,115 @@
 <!-- wp:template-part {"slug":"header"} /-->
 
 <!-- wp:group {"tagName":"main","metadata":{"name":"Main"},"layout":{"type":"default"}} -->
-<main class="wp-block-group"><!-- wp:spacer {"height":"var:preset|spacing|60"} -->
-<div style="height:var(--wp--preset--spacing--60)" aria-hidden="true" class="wp-block-spacer"></div>
-<!-- /wp:spacer -->
+<main class="wp-block-group">
+	<!-- wp:spacer {"height":"var:preset|spacing|60"} -->
```

Comment:
Most of this is reformatting so I could see missing html tags, we were missing one of the ending column tags